openedx/features/edly/utils.py

```diff
@@ -0,0 +1,22 @@
+import waffle
+from django.conf import settings
+from django.contrib.sites.shortcuts import get_current_site
+from openedx.features.edly.models import EdlySubOrganization
+from util.organizations_helpers import get_organizations
+
+def get_enabled_organization(request):
+  """
+  Helper method to get linked organizations for request site.
+
+  Returns:
+      list: List of linked organizations for request site
+  """
+
+  if not waffle.switch_is_active(settings.ENABLE_EDLY_ORGANIZATIONS_SWITCH):
+      organizations = get_organizations()
+  else:
+      current_site = get_current_site(request)
+      edly_organizations = EdlySubOrganization.objects.filter(studio_site=current_site.id)
```

Comment:
`EdlySubOrganization` has a one to one relation with Site. That means that there can only be one studio site against one `EdlySubOrganization`, so we don't need to filter it. you can just do `edly_organization = current_site.studio_site`

Reply:
@zubair-arbi If you could provide your feedback on this?

Reply:
Sounds good.
But we need to note that the openedx `core` apps and `features` apps can be run with both LMS and Studio (CMS) settings.